### PR TITLE
build(version): bump app version to 1.0.2

### PR DESCRIPTION
**Pull request by deorukhkarchaitanya**

### CODE REVIEW

File: ios/blade/Info.plist [hunk 1]
```diff
@@ -17,11 +17,11 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+	<string>1.0.3</string>
```
Comment (dhruvdutt) [inline ios/blade/Info.plist]: Mismatch between gradle version and this? 🤔

Comment (deorukhkarchaitanya) [reply ios/blade/Info.plist]: Should be 1.0.2

File: ios/blade/Info.plist [hunk 2]
```diff
@@ -40,7 +40,13 @@
...
 	</dict>
 	<key>NSLocationWhenInUseUsageDescription</key>
-	<string/>
+	<string></string>
+	<key>UIAppFonts</key>
+	<array>
+		<string>Lato-Bold.ttf</string>
+		<string>Lato-Light.ttf</string>
+		<string>Lato-Regular.ttf</string>
+	</array>
```
Comment (kamleshchandnani) [inline ios/blade/Info.plist]: Do the consumers need to add this as well when they use our DS or when they `link` it will be setup automatically?

Comment (deorukhkarchaitanya) [reply ios/blade/Info.plist]: This will be set up automatically. This diff is also XCode rearranging it.

**Pull request merged**